doc/index.rst

```diff
@@ -7,7 +7,8 @@ Blackbird Quantum Assembly Language
 :Release: |release|
 :Date: |today|
```

Comment:
Not sure why these don't seem to be updated in the (latest) docs: 
https://quantum-blackbird.readthedocs.io/en/latest/

Thought triggering a build via a change in the file (below) might do something, but the date doesn't seem to coincide with other doc updates. 🤔

Reply:
Ah, I think I see why: https://readthedocs.org/projects/quantum-blackbird/

The docs have been failing to build for a long time :( Let me activate PR doc builds for this repo.

Reply:
It seems to be failing with
```
  File "/home/docs/checkouts/readthedocs.org/user_builds/quantum-blackbird/envs/latest/lib/python3.6/site-packages/breathe/node_factory.py", line 5, in <module>
    import sphinx.ext.mathbase
ModuleNotFoundError: No module named 'sphinx.ext.mathbase'
```

Reply:
Needed to bump the `breathe` version and pin `sphinxcontrib-bibtex<2.0.0`. See details [here](https://github.com/michaeljones/breathe/issues/451) and [here](https://github.com/executablebooks/jupyter-book/issues/1137#issuecomment-743994864). Can likely be fixed in a better way, using newer versions, but this makes things work for now.